jormungandr/src/explorer/graphql/mod.rs

```diff
@@ -571,8 +571,64 @@ impl PoolRegistration {
             .collect()
     }
 
-    // TODO: rewards
-    // TODO: keys
+    pub fn operators(&self) -> Vec<PublicKey> {
+        self.registration
+            .operators
+            .iter()
+            .map(PublicKey::from)
+            .collect()
+    }
+
+    pub fn rewards(&self) -> TaxType {
+        TaxType(self.registration.rewards)
+    }
+
+    /// Reward account
+    pub fn reward_account(&self, context: &Context) -> Option<String> {
+        // TODO: What's the best way to show this? As an Address?
```

Comment:
@NicolasDP what do you think of this? reward_account is an `AccountIdentifier`, but that's not in the graphql API at this moment. Is `Address` a good type for something like this or should it be something more specific?